docs/pages/admin-guides/management/security/reduce-blast-radius.mdx

```diff
@@ -17,7 +17,7 @@ Teleport encourages users to practice defense in depth so that every component o
 
 If a user sets up an account to authenticate to their Teleport cluster with only a password, an adversary can gain access to the password using brute-force attacks, person-in-the-middle attacks, or phishing. But even if a user's password is compromised, you can stop an attacker from authenticating with it when they run `tsh login`.
 
-Teleport lets you make it mandatory for a user to enroll an MFA device when they create an account, and to authenticate using that device when they begin a new Teleport session.
+Teleport lets you set the type of MFA device required when they create an account, and to authenticate using that device when they begin a new Teleport session.
```

Comment:
We need more changes here. This reads awkwardly as-is. It basically says "you shouldn't do  X. X is insecure. By the way, it's not possible to do X"

Reply:
I wondered if this section should be removed since it's mandatory. Or perhaps make the mandatory be webauth over OTP?